R/calculate-effect-sizes.R

```diff
@@ -410,6 +411,35 @@ batch_calc_ES <- function(dat,
   
   ES_long_names <- names(ES_ests_long)
   
+  # weights
+  conditions <- dat %>% dplyr::select({{condition}}) %>% unique() %>% dplyr::pull()
+  
+  if (is.null(baseline_phase)) baseline_phase <- conditions[1]
+  
+  if (is.null(intervention_phase)) {
+    intervention_phase <- setdiff(conditions, baseline_phase)[1]
+    if (length(conditions) > 2) {
+      msg <- paste0("The 'condition' variable has more than two unique values. Treating '", intervention_phase, "' as the intervention phase.")
+      warning(msg)
+    }
+  }
+  
+  dat <- 

```

Comment:
Would it work to combine this chunk with the next chunk (that calculates `n_weights`) so that the source data doesn't get modified?